test/test_expanded_weights.py

```diff
@@ -203,6 +203,7 @@ def test_expanded_weights_per_sample_grad_input_no_grad(self, device, dtype, op)
 
             self._compare_ew_and_for_loop_per_sample_grads(op, sample_input, torch.mean)
 
+    @skipIfTorchDynamo("Checking error message doesn't work with dynamo")
     @ops(filter(lambda op: op.supports_expanded_weight, op_db), dtypes=OpDTypes.supported, allowed_dtypes=(torch.double,))
     def test_unsupported_expand_weights(self, device, dtype, op):
```

Comment:
We are testing if the error message matches, but Dynamo compilation probably raise other error message if it's not supported.